airgun/views/host.py

```diff
@@ -193,7 +198,7 @@ class HostsView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[text()='Hosts']")
     export = Text(".//a[contains(@class, 'btn')][contains(@href, 'hosts.csv')]")
     new = Text("//a[contains(text(),'Create Host')]")
-    register = Button("Register Host")
+    register = pf4Button('OUIA-Generated-Button-secondary-2')
     select_all = Checkbox(locator="//input[@id='check_all']")
```

Comment:
I don't see `register = pf4Button('OUIA-Generated-Button-secondary-2')` on my 6.10 satellite.
I tried `Register Host` button on `All Host` page and `Register Content Host` button from Content host page.
@omkarkhatavkar where's this button located?

Reply:
![register_host](https://user-images.githubusercontent.com/3190629/126596703-7925653f-66fb-42e1-af48-7391985cf9b3.png)
As the tests are ran on unused org and location the register host button will be displayed on the blank page. I will also cross check is that something goes away with running multiple tests. I will cross check that condition here.